src/coredump/coredump.c

```diff
@@ -1212,8 +1218,25 @@ static int process_kernel(int argc, char* argv[]) {
 
         if (context.is_journald || context.is_pid1)
                 r = submit_coredump(&context, iovw, STDIN_FILENO);
-        else
+        else {
+                _cleanup_free_ char *process_name = NULL;
+                process_name = filename_escape(context.meta[META_COMM]);
+                if (!process_name)
+                        log_oom();
+                else {
+                        /* per process coredump ratelimit */
+                        r = coredump_ratelimit(process_name, arg_ratelimitinterval, arg_ratelimitburst);
+                        if (r < 0)
+                                log_warning("Failed to process coredump ratelimit");
```

Comment:
coredump_ratelimit() already logs about all its error itself, hence no need to log here again